src/i18n/locales.js

```diff
@@ -1,9 +1,12 @@
 import enUS from './locales/en-US.json'
+import deDE from './locales/de-DE.json'
 
 export const translatedBy = {
   // 'en-US': { github: 'https://github.com/username', name: 'Your Name' },
+  // 'de-DE': { github: 'https://github.com/paulhoenhorst', name: 'Paul Hoenhorst' },
```

Comment:
This needs to be uncommented if you want it to be visibly in the UI 😁 
```suggestion
  'de-DE': { github: 'https://github.com/paulhoenhorst', name: 'Paul Hoenhorst' },
```